README.md

```diff
@@ -56,8 +56,88 @@ pip install -e cloudify-openstack-plugin
   - manager_blueprint
   - properties
 
-* Run the test using `nosetests`:
```

Comment:
worth having handler configuration (for running tests) as its own section (things that are relevant when running tests locally):
- install_manager_blueprint_dependencies
- skip_cleanup
- manager_ip
- properties
- manager_blueprint
- inputs
- handler